encord/objects/common.py

```diff
@@ -117,29 +126,19 @@ def to_dict(self) -> dict:
     def from_dict(cls, d: dict) -> Attribute:
```

Comment:
uf I'm realising how much I was violating the open-close principle!

Reply:
Haha, looking at own old code is always fun :D